src/Runner.php

```diff
@@ -156,6 +157,43 @@ public function execute($argv, $appName = null, $appVersion = null, $output = nu
         return $this->run($argv, $output, $app, $commandFiles, $this->classLoader);
     }
 
+    /**
+     * Return an initialized application loaded with specified commands and configuration.
+     *
+     * This should ONLY be used for testing purposes. Works well in conjunction with Symfony's CommandTester.
+     *
+     * @see https://symfony.com/doc/current/console.html#testing-commands
+     * @see CommandTestertTest
+     * @see CommandTesterTrait
+     *
+     * @param string|null $appName
+     *   Name of the application.
+     * @param string|null $appVersion
+     *   Version of the application.
+     * @param string|array|null $commandFile
+     *   Name of the specific command file, or array of commands, that should be included with the application.
+     * @param \Robo\Config\Config|null $config
+     *   Robo configuration to be used with the application.
+     * @param \Composer\Autoload\ClassLoader|null $classLoader
+     *   Class loader to use. If PHPUnit and the application runner rely on the same autoloader, this can be left null.
```

Comment:
The class loader is only needed if testing Robo plugins. I think this is only rarely going to be needed, but might as well leave it in for those fringe cases.

Reply:
The comment is not correct, though; it doesn't matter if PHPUnit and the application have the same or different autoloader, the plugin tests won't work unless this reference is provided, and the classloader isn't needed unless loading plugins.